docs/source/glossary.rst

```diff
@@ -4,200 +4,190 @@ OpenMS Glossary
 A glossary of common terms used throughout OpenMS documentation.
 
 .. glossary::
-  :sorted:
+    :sorted:
 
-  LC-MS
-  LCMS
-    Liquid Chromatography-Mass Spectrometry. 
+    LC-MS/MS
+        See :term:`Liquid chromatography` and :term:`MS2`.
 
-  Liquid chromatography
-    An analytical technique used to separate molecules of interest.
+    LC-MS
+        Liquid chromatography-mass spectrometry, see :term:`Liquid chromatography` and :term:`mass spectrometry`.
 
-  mass spectrometry
-    An analytical technique used to identify and quantify molecules of interest.
+    Liquid chromatography
+        An analytical technique used to separate molecules of interest.
 
-  peptides
-    A short chain of amino acids.
+    Mass spectrometry
+        An analytical technique used to identify and quantify molecules of interest.
 
-  FASTA format
-    A text-based format for representing nucleotide or amino acid sequences.
+    Peptides
+        A short chain of amino acids.
 
-  octadecyl (C18)
-    An alkyl radical C(18)H(37) derived from an octadecane by removal of one hydrogen atom.
+    FASTA format
+        A text-based format for representing nucleotide or amino acid sequences.
 
-  Mass
-    Mass is a measure of the amount of matter that an object contains. In comparison to often used term weight, which is a measure of the force of gravity on that object.
+    Octadecyl (C18)
```

Comment:
capitalization and inclusion of abbreviations actually looks like the least efficient choice for glossary terms.
@timosachsenberg @tjeerdijk See discussion in #331

Reply:
Maybe like so: https://style.mla.org/formatting-a-glossary/

Reply:
Yes. But my main concern is not styling, but usage. If you capitalize the terms, you need to write :term:`non-capitalize <Non-capitalize>` all the time. When in English you need the lower case 95% of the time.

Reply:
Same goes for abbreviations. If you add "mass spectrometry (MS)" as a term you can neither use :term:`mass spectrometry` nor :term`MS` separately.
I want to balance the amount of terms in the glossary and the number of times you need to add an alternative text. Therefore we need rules.

By the way, you can have multiple terms for the same description. Not sure why you removed that idea.

Reply:
see #331 unfortunately no one has an opinion there.

Reply:
Unless its a proper noun, then it needs to be capitalized anyway. So using lowercase for all except proper nouns should solve the issue of writing it twice and adheres to the conventions.

Reply:
yes "OpenMS" or "Bayesian" would be fine.

Reply:
or things that mainly exist as acronyms

Reply:
I tried to find a reference on how to use abbreviations in a glossary, but haven't really found any. I suggest we do not use abbreviations in the keyword itself, but only the full name, e.g. mass spectrometry instead of MS.

Reply:
I agree. But would you add MS as a second possible term? Often you introduce the abbreviation and then only use the abbreviation throughout the rest of the text. Then you would need to specify the full name in the "term" role.
I am ok with it, but we have to be aware of the consequences.

Reply:
How does this show in the pop up?

Reply:
Don't know :)

Reply:
I guess with the term that was used in the :term: that triggered the pop up.
Of course you have to start your descriptions with "Mass spectrometry (MS) is ..." as I changed in my previous PR.

Reply:
It doesn't look great in the glossary itself and from the formatting it lalso ooks like it will always be registered as two separate terms https://pyopenms.readthedocs.io/en/latest/glossary.html?highlight=lcms#term-LC-MS.

Reply:
Personally, I prefer it over many "see XYZ" entries. That's why I wanted to trigger a discussion with opinions.

Reply:
How does it look like in the popup. Seems like it is never referenced...

Reply:
Not many glossary terms are actually linked right now.